features/build_runner/fastlane_build_runner.rb

```diff
@@ -58,19 +58,37 @@ def run
       # this only takes a few ms the first time being called
       Fastlane.load_actions
 
-      # Load and parse the Fastfile
-      # TODO: This won't work for now, as it is evaluating to the local CI fastlane.
       fast_file_path = self.project.local_fastfile_path
       fast_file = Fastlane::FastFile.new(fast_file_path)
 
       begin
+        # TODO: I think we need to clear out the singleton values, such as lane context, and all that jazz
         # Execute the Fastfile here
         logger.info("starting fastlane run lane: #{self.lane} platform: #{self.platform}, params: #{self.parameters} from #{fast_file_path}")
+
+        build_success = true
+        # Attach a listener to the output to see if we have a failure. If so, this build failed
+        self.add_listener(proc do |row|
```

Comment:
Wow, this is solid 😍

Reply:
Only made possible by your `listener` idea 💯